File: 800/1986A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t; // number of test cases

    while (t--) 
    {
        int a, b, c;
        cin >> a >> b >> c; // three integers

        int mini = min({a, b, c}); // minimum of the three
        int maxi = max({a, b, c}); // maximum of the three

        int ans = INT_MAX; // initialize with large value

        // Try all possible meeting points between min and max
        for (int i = mini; i <= maxi; i++) 
        {
            int dist = abs(i - a) + abs(i - b) + abs(i - c);
            ans = min(ans, dist);
        }

        cout << ans << "\n"; // print the minimum distance
    }

    return 0;
}

File: 800/1986A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int a = sc.nextInt();
            int b = sc.nextInt();
            int c = sc.nextInt();
            
            int mini = Math.min(a, Math.min(b, c));
            int maxi = Math.max(a, Math.max(b, c));
            
            int ans = 100;
            for(int i=mini;i<=maxi;i++)
            {
                int dist = Math.abs(i-a)+Math.abs(i-b)+Math.abs(i-c);
                ans = Math.min(ans, dist);
            }
            
            System.out.println(ans);
        }
    }
}